wbce/framework/I.php

```diff
@@ -283,7 +283,7 @@ public static function addUrlToken($sToken = "", $sURL = "") {
      */
     public static function doFilter($Content) {
        $result = I::getInstance()->doFilter($Content);
-       I::$oInstance = NULL;
+       #I::$oInstance = NULL;
        return $result;

```

Comment:
With the more elegant solution we could revert this function to the original oneliner

Reply:
You're absolutely right, Martin.